runtime/tchannel_interfaces.go

```diff
@@ -43,5 +43,5 @@ type TChannelCaller interface {
 type TChannelHandler interface {
 	// Handle should read the request from the given reqReader, and return the response struct.
 	// The arguments returned are (whether there was an application error, response headers, result struct, unexpected error)
```

Comment:
The interface description might need an amendment here.

Reply:
Amended